base/kra/shared/conf/CS.cfg

```diff
@@ -343,13 +343,13 @@ logAudit.fileName=[PKI_INSTANCE_PATH]/logs/[PKI_SUBSYSTEM_TYPE]/access
 logError.fileName=[PKI_INSTANCE_PATH]/logs/[PKI_SUBSYSTEM_TYPE]/error
 oidmap.auth_info_access.class=org.mozilla.jss.netscape.security.extensions.AuthInfoAccessExtension
 oidmap.auth_info_access.oid=1.3.6.1.5.5.7.1.1
-oidmap.challenge_password.class=com.netscape.cms.servlet.cert.scep.ChallengePassword
+oidmap.challenge_password.class=org.mozilla.jss.netscape.security.x509.ChallengePassword
```

Comment:
@ladycfu -- I thought we discussed that we wanted to remove SCEP's `ChallengePassword` from KRA/OCSP/...'s configs?

Reply:
+1

Reply:
No.  I remember I said I'll keep them for now as even though it is currently only used for SCEP, it might be used for something else later.  And we don't need to sort it out at this point.  I thought maybe it was Endi who said he agreed?